newsroom/cards/cards.py

```diff
@@ -40,7 +40,9 @@ class CardsResource(newsroom.Resource):
         'dashboard': {
             'type': 'string',
             'required': True,
-        }
+        },
+        'original_creator': {'type': 'ObjectId'},
```

Comment:
For all the `original_creator` and `version_creator`, use `newsroom.Resource.rel('users')` for the field schema.